source/edge-server.txt

```diff
@@ -0,0 +1,144 @@
+:template: product-landing
+:hidefeedback: header
+:noprevnext:
+
+.. _edge-server:
+
+=================
+Atlas Edge Server
+=================
+
+.. toctree::
+   :titlesonly:
+   :hidden:
+   
+   Configure Edge Server </edge-server/configure>
+   Connect to Edge Server </edge-server/connect>
+   Use MongoDB APIs with Edge Server </edge-server/mongodb>
+
+.. introduction::
+
+   You have a warehouse, retail store, air-gapped laboratory, or another 
+   premises where devices should be able to seamlessly sync their data.
+   Those locations may or may not be online at any given time. Lack of a 
+   network connection shouldn't prevent the devices from sharing data
+   with each other.
+
+   Edge Server runs on-premises and handles sync between local devices,
+   and bi-directional sync between the Edge Server and Atlas.
+
+   Edge Server is currently in private preview. If you are interested in 
+   previewing the Atlas Device Sync Edge Server, complete this form.
```

Comment:
Still feels like a big disconnect between "...complete this form." and the CTA button label

the existing [Atlas Device Sync: Edge Server](https://www.mongodb.com/docs/atlas/app-services/sync/app-builder/edge-server/#:~:text=is%20currently%20in-,Private%20Preview,-.) page has it worded slightly differently (not much better, tbh)
```suggestion
   previewing the Atlas Device Sync Edge Server, complete this form to let us know.
```

Reply:
I left the "to let us know" off because it seemed redundant and unnecessary. I'll tweak the wording a little bit to make it clearer what's happening. But the CTA is changing for .local London, so this is only going to be live for a short time before it changes.